src/main/java/net/openhft/chronicle/bytes/internal/ChunkedMappedFile.java

```diff
@@ -84,7 +85,7 @@ public static void warmup() {
         try {
             Jvm.setExceptionHandlers(Slf4jExceptionHandler.ERROR, null, null);
 
-            final Path path = Files.createTempDirectory("warmup");
+            final Path path = Files.createTempDirectory(Paths.get("warmup_dir"), "warmup");
```

Comment:
Should it be configurable by property? Will it be allocated in tmpdir or curdir?